### PR TITLE
Read keys from pbf

### PR DESCRIPTION
**Pull request by agila5**
Reprex: 

``` r
# packages
library(osmextract)
#> Data (c) OpenStreetMap contributors, ODbL 1.0. https://www.openstreetmap.org/copyright.
#> Check the package website, https://docs.ropensci.org/osmextract/, for more details.

# example
west_yorkshire <- oe_get("West Yorkshire", download_only = TRUE, skip_vectortranslate = TRUE)
#> The input place was matched with: West Yorkshire
#> The chosen file was already detected in the download directory. Skip downloading.

# read keys
head(oe_get_keys(west_yorkshire))
#> [1] "lit"           "addr:postcode" "lcn"           "maxspeed"     
#> [5] "sidewalk"      "oneway"
```

<sup>Created on 2021-04-06 by the [reprex package](https://reprex.tidyverse.org) (v1.0.0)</sup>

### CODE REVIEW

File: R/vectortranslate.R [hunk 1]
```diff
@@ -383,23 +383,21 @@ get_ini_layer_defaults = function(layer) {
...
-#'   column.
+#'   formats is governed by a `CONFIG` file that lists which tags must be
+#'   explicitly added to the `.gpkg` file, while all the other keys are
+#'   automatically stored using an `other_tags` field with a syntax compatible
+#'   with the PostgreSQL HSTORE type. This function can be used to display the
+#'   names of all keys stored in the `other_tags` field.
 #'
-#'   You can also use the `hstore_get_value()` function from GDAL to extract one
-#'   particular tag from an existing `.gpkg` file. Check the introductory
+#'   The `hstore_get_value()` function can be used inside the `query` argument
```
Comment (Robinlovelace) [inline R/vectortranslate.R:399]: I think a function like `oe_get_values()` could be good if we can manufacture the query. Just an idea, may be hard to implement.

**Review by Robinlovelace — COMMENTED** [review]
Looks good, made a few suggested changes in a PR.

**Comment by Robinlovelace** [comment]
This is good and a definite improvement that enables users to 'see' the available tags from a large pbf file before reading it in. That will be really useful. However, I don't think it provides an easy way to 'see' the values associated with a particular key. Building on the discussion topic code:

```r
d = oe_get("isle of wight", extra_tags = "railway")
table(d$railway)
#> 
#>    abandoned construction      disused    miniature     proposed         rail 
#>          134            2            3           12            3           73 
#>        razed 
#>           29
```

I think a function along the lines of `oe_get_values()` could be very useful. It could work like this:

```r
oe_get_value("pbf_file.pbf", key = "highway", layer = "lines")
# returns the unique highway values without translating to gpkg
```

What I'm not sure is whether it's possible/easy to get values associated with keys not in the default .ini files, e.g.:

```r
oe_get_value("pbf_file.pbf", key = "railway", layer = "lines")
```

Links with: https://github.com/ropensci/osmextract/discussions/188

**Comment by agila5** [comment]
I agree with all your comments but I would work on that function on a separate PR (probably during the weekend).

**Review by Robinlovelace — APPROVED** [review]
Looking good

**Pull request merged**